src/herbivoresAndCarnivores.js

```diff
@@ -1,17 +1,45 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+  }
```

Comment:
I think it would be better to add any animal to the alive table in the animal parent class. Otherwise it is not an animal and mustn't extend the parent one.
```suggestion
    this.health = 100;
    
    Animal.alive.push(this);
  }
```